### PR TITLE
FirstFound Updates - Homepage

### PR DESCRIPTION
**Pull request by ahymanFirstFound**
Updated title tags and description tags

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,9 +1,7 @@
 ---
-title: Michael Westcott Films
+title: Wedding Videography | Aberdeen, Glasgow, Edinburgh, Scotland
```
Comment (kscott7) [inline index.html]: This title is a duplicate of the title on the Weddings page?

Comment (ahymanFirstFound) [reply index.html]: Wedding page: 
<title>Wedding Videos, Videographer, Filmmaker based in Aberdeen, Scotland</title>
Home page: 
<title>Wedding Videography | Aberdeen, Glasgow, Edinburgh, Scotland</title>

Comment (kscott7) [reply index.html]: Ok, then can we keep "Michael Westcott Films |"?
Proposed...
title: Michael Westcott Films | Wedding Videography - Aberdeen, Glasgow, Edinburgh, Scotland

Comment (ahymanFirstFound) [reply index.html]: On that I will have to say no, only down to the fact that Google or any search engine reads the first bit of information and marks it as important. We could do:
title: Wedding Videography, Aberdeen, Glasgow, Edinburgh, Scotland | Michael Westcott Films

Comment (kscott7) [reply index.html]: Ok, just leave as is then 👍

**Comment by ahymanFirstFound** [comment]
Hi Kieron, 

I have made all the requested changes, please can you put them live.

Many thanks,
Ashley

**Comment by kscott7** [comment]
I updated the title to the one you suggested above...hope this is ok.

**Pull request merged**